src/lib.rs

```diff
@@ -521,17 +487,58 @@ pub enum LocParseError {
     SecondTabulatorMissing,
 }
 
-/// A brief in-app documentation.
-///
-/// This function will write a brief usage information, including a short
-/// introduction to the meaning of the configured `options`, to `stdout`.
-pub fn usage(options: &getopts::Options) {
-    let description = "Parses the Git history.";
-    let name = "commit-analyzer";
-    let synopsis = "[OPTIONS]";
-
-    println!(
-        "{name}.\n{description}\n\n{}",
-        options.usage(&format!("Usage: {name} {synopsis}"))
-    );
+/// Parses the Git history.
+#[derive(Parser, Debug)]
+#[clap(author, version, about, long_about = None)]
```

Comment:
There is no `author` field configured in the `Cargo.toml`.  Please add it or remove the `author` derivation.

Reply:
Done 👍